apps/catalog-admin/components/code-list-editor/index.tsx

```diff
@@ -236,7 +236,7 @@ export const CodeListEditor = ({ dbCodeList }: Props) => {
     // Not itself, not child, null to remove parent
     return (
       codes
-        ?.filter((code: Code) => code.id !== currentCode.id && code.parentID !== currentCode.id)
+        ?.filter((code: Code) => code.id !== currentCode?.id && code.parentID !== currentCode.id)
```

Comment:
Er det ikke bedre å starte med nullsjekk for currentCode, og returnere tom liste hvis den ikke er satt?

Det virker på som at potensialet for at currentCode ikke har verdi gjør filteret noe uoversiktelig